### PR TITLE
Limit access ai

### PR DESCRIPTION
**Pull request by evgeniya-aira**
Added a new accessAi property to the User's Account (Profile) data.

**Details**
Introduced the accessAi property to the User's Account (Profile) model.
This property is intended to manage Access AI permissions to different user's profiles.

**Example**:
A response example including the new `accessAi` property can be found in the following [issue](https://linear.app/airaio/issue/PLAT-269/limit-access-ai-platform-add-support-for-explorer-app#comment-b851829b).

### CODE REVIEW

File: lib/src/models/account.dart [hunk 1]
```diff
@@ -15,13 +15,35 @@ enum AccountType {
...
+  final String value;
+
+  const AccessAi(this.value);
+
+  /// Gets the [AccessAi] from the given value.
+  ///
+  /// If the value is not found, [noAccess] is returned.
+  static AccessAi fromValue(String? value) {
+    return AccessAi.values
+        .firstWhere((e) => e.value == value, orElse: () => AccessAi.noAccess);
```
Comment (ming-chu) [inline lib/src/models/account.dart:32]: I am not sure if we want to make it upper case `.toUpperCase()` before compare.

**Review by ming-chu — APPROVED** [review]
LGTM

File: lib/src/models/account.dart (hunk 1 above)
Comment (ming-chu) [reply lib/src/models/account.dart:32]: image

if the [API](https://linear.app/airaio/issue/PLAT-269/limit-access-ai-platform-add-support-for-explorer-app#comment-b851829b) always return upper case, it should be fine.

Comment (evgeniya-aira) [reply lib/src/models/account.dart:32]: Screenshot 2024-08-20 at 9 14 02 AM
It looks like it's in uppercase in the [API](https://github.com/aira/platform/blob/master/core/server/AiraPlatformData/src/main/java/io/aira/data/model/Account.java)

**Pull request merged**